### PR TITLE
[weekly-update] Add weekly updates

### PR DESCRIPTION
**Pull request by yosuke-furukawa**
@nodejs/evangelism 
Please review this weekly update.

### CODE REVIEW

**Comment by Trott** [comment]
Is it too late to include v3.3.0 of io.js?

File: weekly-updates/weekly-update.2015-08-28.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* Some problems with unreferenced timers running during `beforeExit` are still to be resolved. See [#1264](https://github.com/nodejs/io.js/issues/1264).
+* Surrogate pair in REPL can freeze terminal. [#690](https://github.com/nodejs/io.js/issues/690)
+* `process.send()` is not synchronous as the docs suggest, a regression introduced in 1.0.2, see [#760](https://github.com/nodejs/io.js/issues/760).
+* Calling `dns.setServers()` while a DNS query is in progress can cause the process to crash on a failed assertion. [#894](https://github.com/nodejs/io.js/issues/894)
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/io.js/issues/1435).
+
+
+### We need Node.js v4 evangelists
+
+* Node.js v4 is released soon. We need evangelists for Node.js v4. If you have an interest to evangelize, send your information in [this issue](https://github.com/nodejs/node/issues/2633).
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-08-28.md]: Nit: `send your information` -> `provide your information`

File: weekly-updates/weekly-update.2015-08-28.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+If you have spotted or written something about Node.js and io.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [NodeConf EU](http://nodeconf.eu/) tickets are on sale, September 6th - 9th at Waterford, Ireland
+* [nodeday](http://nodeday.com/) tickets are on sale, September 10th at Karnataka, India
+* [Node.js Italian Conference](http://nodejsconf.it/) tickets are on sale, October 10th at Desenzano - Brescia, Italy
+* [EmpireNode](http://2015.empirenode.org/), October 23rd at New York, US.
+* [JSConf CO](http://www.jsconf.co/), October 16th - 17th at Ruta N, Medellin
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-08-28.md]: Nit: All the other event listings mention the country they take place in, even if it's explicit in the title of the conference. So maybe mention that this one is in Colombia?

**Comment by yosuke-furukawa** [comment]
Thank you @Trott !!

I fixed all your comments. I will publish weekly update asap.

**Pull request merged**